notifications/notifications.go

```diff
@@ -0,0 +1,14 @@
+// Package notifications contains subpackages related to Icinga Notifications.
+//
+// The event package describes the central Event type used to describe Icinga Notifications events.
```

Comment:
Avoids using the same verb twice in a sentence.
```suggestion
// The event package defines the central Event type used to describe Icinga Notifications events.
```